example/spec/helpers/reporters/junitReporter.js

```diff
@@ -0,0 +1,7 @@
+
+const reporters = require('jasmine-reporters');
+const junitReporter = new reporters.JUnitXmlReporter({
+  savePath: process.env.JUNIT_DIR || '/tmp',
```

Comment:
This dumps junit tests locally to /tmp.   We could dump it to /dev/null.... 🤔

Reply:
will we have sufficient access to the runner machine to make use of these logs?

Reply:
@ifestus  in the CI setup, JUNIT_DIR is defined, which results in the .xml results being dumped to that directory, resulting in the JUNIT parsing step picking up the tests and utilzing them. 

The '/tmp' output is when running integration tests on a development machine.